pkg/cloudprovider/fake/controller-kwok/charts/templates/aggregate-clusterrole.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
How much of this chart can be dropped since there is no need for backwards compatability? e.g. Can we drop the webhooks, the configmap logging, etc.?

Reply:
Good callout, I hadn't thought about what was actually needed vs what was core and what wasn't. We can probably drop webhooks and configmap logging, but can take another pass at it in a separate PR, if that makes it easier to review.